src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,8 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public boolean isMutating() {
+        return true;
```

Comment:
This method should have a header comment. Only then users of the overridden method will know how it exactly behaves.